### PR TITLE
Create README

### PR DESCRIPTION
**Pull request by emmaknor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,74 @@
...
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+A user will first sign up for an account where they are able to enter their calorie, macro, and water goals. Upon log in the user is able to submit entries and then will be updated on their progress and recieve motivation based on their daily activity. Our hope is to  support people in achieving their weight loss goals in a fun, motivational way.
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+## Contributors:
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+[Scott Guinn](https://github.com/Scott-Guinn) - Architecture Owner
```
Comment (Scott-Guinn) [inline README.md]: Can we call me a Software Architect? Or "Lead Software Architect"? I think it makes more sense than Architecture Owner

File: README.md [hunk 2]
```diff
@@ -1,34 +1,74 @@
...
 
-## Learn More
+[Ciaddie Rutelionis](https://github.com/Ciaddie) - Product Manager
 
-To learn more about Next.js, take a look at the following resources:
+[Taylor Smart](https://github.com/taylorsmart) - UI Owner
```
Comment (Scott-Guinn) [inline README.md]: Maybe same for this, "Lead UI Designer"

File: README.md (hunk 1 above)
Comment (emmaknor) [reply README.md]: sure thing

File: README.md (hunk 2 above)
Comment (emmaknor) [reply README.md]: cool will change - will add dennis as co lead ui designer as well

File: README.md [hunk 3]
```diff
@@ -1,34 +1,74 @@
...
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+[Oren Nelson](https://github.com/nohren)
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+[Ben Kennedy](https://github.com/benkennedy98)
+
+
+## Tech Stack
+NVision Nutrition utilizes React as the front-end framework and a Node server connected to a PostgreSQL database on the back-end configured with Next.js.
```
Comment (Scott-Guinn) [inline README.md]: React is technically a library not a framework.

Comment (Scott-Guinn) [reply README.md]: but we can expand on this section much more once we actually build the thing

File: README.md (hunk 2 above)
Comment (taylorsmart) [reply README.md]: Formally, they would be 
UX Designer - me
UI Designer - Dennis

**Review by Scott-Guinn — APPROVED** [review]
Looks great. Thanks for putting this together guys! Made some comments. Obviously we'll need to go back and update this as the project goes on/finishes.

**Review by dennismejia — APPROVED** [review]

**Pull request merged**